### ISSUE TITLE
about date

### ISSUE THREAD

**Issue by NLP1r** [fdulcg/Chinese-Named-Entity-Recognition#1]
Hi，可以请教一个问题吗？想请教一下data
文件夹内三个文件sighan.train、sighan.test、sighan.dev。前两个分别是训练集和测试集，请问一下最后一个代表什么？和训练集有什么区别吗？求指教。

**Comment by NLP1r** [comment]
Hi, can I ask a question? Would like to ask about dataThree files in the file sighan.train, sighan.test, sighan.dev The first two are training set and test set, what is the last representative of what? What is the difference between a training set and a training set? Seek advice.
